.travis/check_relevant_docs.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```

Comment:
If we would like to keep this, this change should be done against dev-kit

Reply:
no was my fault while mergin